index.js

```diff
@@ -148,8 +148,12 @@ SlackBot.prototype.buildRouter = function () {
   return function (event, context, callback) {
     var foundCommand;
     var builtEvent = event;
-    builtEvent.body = qs.parse(builtEvent.body);
 
+    if (builtEvent.body === 'ping') {
+      return context.succeed('Ok');
+    }
```

Comment:
Will this exit fast enough? Calculations show that pinging every 5 minutes for a full day would cost $0.05/day ($17.55/year) for a lambda running with 1024MB of memory. Is that too much for the number of slackbots we're planning to use this with?